### PR TITLE
Improve readability of built-in components legacy arguments

### PR DESCRIPTION
**Pull request by bertdeblock**
Before:
Screenshot 2021-12-16 at 08 15 18

After:
Screenshot 2021-12-16 at 08 15 31

### CODE REVIEW

File: content/ember/v3/ember-built-in-components-legacy-arguments.md [hunk 1]
```diff
@@ -7,29 +7,34 @@ since: '3.27'
...
+* `@models`
+* `@query`
+* `@replace`
+* `@disabled`
+* `@current-when`
+* `@activeClass`
+* `@loadingClass`
+* `@disabledClass`
+
+### `<Input>`
```
Comment (locks) [inline content/ember/v3/ember-built-in-components-legacy-arguments.md:23]:
```suggestion
#### `<Input>`
```

File: content/ember/v3/ember-built-in-components-legacy-arguments.md [hunk 2]
```diff
@@ -7,29 +7,34 @@ since: '3.27'
...
+
+* `@type`
+* `@value`
+* `@checked`
+* `@insert-newline`
+* `@enter`
+* `@escape-press`
+
+### `<Textarea>`
```
Comment (locks) [inline content/ember/v3/ember-built-in-components-legacy-arguments.md:32]:
```suggestion
#### `<Textarea>`
```

**Review by locks — CHANGES_REQUESTED** [review]
Thanks for the help! I'm not sure `h3`s work because the deprecation guide title is already an `h3`, so I left github suggestions to make them `h4`s instead. Let see if that shows up nicely!

**Comment by bertdeblock** [comment]
@locks Updated!

**Review by jaredgalanis — APPROVED** [review]
Looks great! Thank you 👍

**Pull request merged**